tests/meta/orders/test_meta_orders_api.py

```diff
@@ -321,8 +323,8 @@ def test_submit_unique_sample_case_name(
     sample: MipDnaSample
     for sample in order_data.samples:
         case_id = sample.family_name
-        customer_obj = store.customer(order_data.customer)
-        assert not store.find_family(customer=customer_obj, name=case_id)
+        customer: Customer = store.get_customer_by_customer_id(customer_id=order_data.customer)
+        assert not store.find_family(customer=customer, name=case_id)
```

Comment:
`find_family(customer=customer)` and `get_customer_by_customer_id(customer_id=order_data.customer)`

Should we refactor `order_data ` to hold `order_data.customer_internal_id` instead of `order_data.customer` ?

Reply:
Not here and now